aptos-move/vm-genesis/src/lib.rs

```diff
@@ -1162,6 +1298,37 @@ pub struct VestingPoolsMap {
     pub period_duration_in_seconds: u64,
 }
 
+#[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq, Serialize, Deserialize)]
+pub struct MultiSigAccountWithBalance {
+    pub owner: AccountAddress,
+    pub additional_owners: Vec<AccountAddress>,
+    pub multisig_address: AccountAddress,
```

Comment:
Not sure why this field `multisig_address:AccountAddress` field is required since you do not seem to be using it anywhere. The functions in `genesis.move` does not need this field since the address for the multisig account is created automatically and you receive the vector of addresses as a return value.

Reply:
@supra-yoga , as discussed, please document cross repository dependency properly.

Reply:
That's right, the reason for this field to exist is for hashing it in the SMR-Moonshot and make sure that we are not supplying duplicate data to the Genesis encoder functions. The genesis tx encoder functions need to rewritten in the future to mitigate this.